oereb_client/views/index.py

```diff
@@ -49,6 +49,9 @@ def validate_(self):
         if not isinstance(self.config_.get('availability'), dict):
             raise ConfigurationError('Missing "availability" configuration')
 
+        if not isinstance(self.config_.get('mask_surrounding'), dict):
```

Comment:
I think the config of masking of the surroundings should be optional. I guess some cantons will not use it.